README.md

```diff
@@ -55,13 +55,13 @@ agenda.define('delete old users', function(job, done) {
   User.remove({lastLogIn: { $lt: twoDaysAgo }}, done);
 });
 
-agenda.on('ready', function() {
+agenda.on('ready', async function() {
```

Comment:
Putting this `async` here feels weird. Perhaps instead of relying on `agenda.on('ready')` we should just bake that into `agenda.start()` so clients would just do

```javascript
await agenda.start();

agenda.every('3 minutes', 'delete old users');
// ...
```

Reply:
Oh good point. We don't really even need ready now that we return a promise from `agenda.start()`, right?